### ISSUE TITLE
Create new sendData(byte[] bytes) method

### ISSUE THREAD

**Issue by ritdaw** [DeveloperPaul123/SimpleBluetoothLibrary#18]
It would be useful to have a method with a bytes[] parameter seeing as you already have a corresponding write(byte[] bytes) method in BluetoothUtility.class.

```
    public void sendData(byte[] bytes) {
        if(this.bluetoothSocket != null && this.bluetoothSocket.isConnected() && this.connectedThread != null) {
            this.connectedThread.write(number);
        }
    }
```

**Comment by ptsouchlos** [comment]
Good point, I'll add that as soon as I can. 

BTW may I ask how you made your Github profile image?

**Comment by ritdaw** [comment]
Thanks.
Someone made it for me for a small fee.

**Comment by ptsouchlos** [comment]
Commit 2282a47967a066d89fa298deeae6b1ee6de4c095 closes #18
